behat-tests/features/bootstrap/FeatureContext.php

```diff
@@ -369,30 +402,21 @@ public function assertRegionElementHasClass($selector, $region, $class) {
   }
 
   /**
-   * @When I wait for messagebox to close
```

Comment:
Hmm - was this really not being used anywhere?  I don't see the calls to it... but I remember the need for it was because the just checking ajax completion isn't enough to have the message box closed.  Plus sometimes we want to see something that happens before the message box closes but the drivers see ajax as still being active when it is on screen.